libff/algebra/fields/fp6_3over2.hpp

```diff
@@ -36,6 +36,14 @@ template<mp_size_t n, const bigint<n> &modulus> class Fp6_3over2_model
     typedef Fp_model<n, modulus> my_Fp;
     typedef Fp2_model<n, modulus> my_Fp2;
 
+    // from scipr-lab (VV)
```

Comment:
same comment about `(VV)` and comments at end of line.

Reply:
Done.